languages/fsharp/exercises/concept/discriminated-unions/.meta/design.md

```diff
@@ -0,0 +1,50 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept of Discriminated Unions in [F#][discriminated-unions].
+
+## Learning objectives
+
+- Know what discriminated unions are.
+- Know how discriminated unions are different from enums.
+- Know how to define a discriminated union, with and without data.
+- Know how to use `match` to pattern match on discriminated unions.
+- Know what pattern matching is.
+- Know about constant, identifier and wildcard patterns.
+- Know how to apply guards to patterns.
+- Know about exhaustiveness checking in pattern matching.
+
+## Out of scope
+
+- Recursive discriminated unions.
+- Single type wrapper discriminated unions.
+- Active patterns.
+- Adding members to discriminated unions.
+- `function` pattern shorthand notation.
+
+## Concepts
+
+The Concepts this exercise unlocks are:
+
+- `discriminated-unions-basic`: know what discriminated unions are; know how discriminated unions are different from enums; know how to define a discriminated union, with and without data; know how to use `match` to pattern match on discriminated unions.
+- `pattern-matching-basic`: know what pattern matching is; know about constant, identifier and wildcard patterns; know how to apply guards to patterns; know about exhaustiveness checking in pattern matching.
+
+## Prequisites
+
+This exercise's prerequisites Concepts are:
+
+- `functions-basic`: defining functions and scoping.
+- `integers-basic`: used as one of the DU's data values.
```

Comment:
This is the first and only place you use DU. Does it make sense to either eliminate this or put discriminated unions in parenthesis after the value to make it explicit what this is. Then DU could be used later without making it explicit.